### DISCUSSION TITLE
yarn install fails during deploy

### DISCUSSION THREAD

**Discussion by manoj-ca** [vercel/community#2539] [Help] [answered]
### Summary

Here is a screenshot of the failure. How do I fix this?
![yarnInstallFail](https://github.com/vercel/community/assets/99002958/754da124-cadc-4ad6-85f3-30090028a65b)

**Comment by amyegan** [comment, marked as answer]
It looks like there is a problem in the `yarn.lock` file. If you delete and regenerate the file, do you get the same error on new builds?

https://stackoverflow.com/questions/70862902/getting-invalid-value-type-error-when-trying-to-execute-yarn-in-my-react-rails

**Reply by manoj-ca** [reply]
Thanks! That worked!
